jsonmerge_utils.py

```diff
@@ -5,10 +5,20 @@
 @author: GRENTOR
 """
 import os
+import logging
 #from tqdm import tqdm
 from merge_files import Merge
-
-
+logger = logging.getLogger(__name__)
+def format_logger():
+    logger.setLevel(level=logging.INFO)
+    if not logger.handlers:
+        # Prevent logging from propagating to the root logger
+        logger.propagate = 0
+        log_formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        stream_handler = logging.StreamHandler()
+        stream_handler.setFormatter(log_formatter)
+        logger.addHandler(stream_handler)
+format_logger()

```

Comment:
this function should be called from the main() function within the try block
like this:
```
try:
    format_logger()
    data_dir = input('Folder Path: ')
```